docs/contributing/docs-writing-process.md

```diff
@@ -0,0 +1,99 @@
+
+4. In the event there is no information available to write a greenfield doc after trying all the above steps, such as for a new integration, try interviewing Gatsby core team members to help to produce an outline and content tips. See [Pair Programming program](/contributing/pair-programming/).
+
+5. **For Gatsby inkteam members:** You can also search in the Gatsby Slack internally for related information. It’s possible other team members have discussed the issue, or even compiled trouble points into a doc. Look and ask around as part of your discovery work.
+
+After you’ve collected supporting information, you must produce original writing to be accepted in the Gatsby docs. Copying other blog posts, materials, or Gatsby team member interviews word-for-word without attribution is not acceptable or allowed. Furthermore, direct quotes from interviews are also _almost never_ effective for guides, recipes, or tutorials.
+
+Rather, the best way to write greenfield docs is with new text that explains the concepts and ideas you uncovered. With research and demos supporting your writing, can speak more legitimately to use cases and common troubleshooting. Phrasing information in alternative ways for different skill levels is a bit like avoiding usage of the same word or phrase in its definition: if a reader didn’t understand it the first time, they may be more likely to understand it with an alternative explanation.
+
+## Producing learning materials
+
+Create demo sites and examples to provide more authoritative material that supports developer experience. When relevant, **functioning source code is a requirement to writing docs** that truly educate users. Source code examples can also be linked from a doc as an additional resource. Include testing as part of your source code to ensure it is robust and stands the test of time.
+
+Follow the [docs templates](/contributing/docs-templates/) to ensure you’re producing content in the right format for its purpose.
+
+Use the [Markdown syntax doc](/docs/mdx/markdown-syntax/) to understand your options for formatting text with Markdown, and follow [accessibility recommendations](/docs/making-your-site-accessible/#how-to-improve-accessibility) for [heading levels](/contributing/docs-contributions/#headings) and image alt text.
+
+Run the docs site locally to check formatting and functionality. There are instructions in the [contributing docs](/contributing/docs-contributions/).
+
+Produce images and rich media following a consistent specification, which will be published on this site soon.
+
+Refer to the [Gatsby style guide](/contributing/gatsby-style-guide/) to ensure your PR will be accepted.
+
+## Submitting a pull request
+
+When submitting work to fulfill an issue, team members are allowed to request feedback in a Google Doc for comments inline. However, this format does not count toward a story deadline internally: only a PR does.
```

Comment:
Should this be part of an internal doc versus an external one? Or if it should be here, perhaps it can go after the information in this section that applies to all contributors?

Reply:
I debated on whether to include this here at all; I think it could go in the Learning handbook instead. Whether anyone actually reads it is another story 🤷‍♀